packages/components/organisms/f-checkout/src/store/tests/checkoutAnalytics.module.test.js

```diff
@@ -426,38 +426,23 @@ describe('CheckoutAnalyticsModule', () => {
                 };
 
                 // Act
-                trackLowValueOrderExperiment(mockedResponseHeaders);
+                trackLowValueOrderExperiment({ rootState, dispatch }, mockedResponseHeaders);
 
                 // Assert
                 expect(window.dataLayer).toContainEqual(expected);
             });
 
             it('should not `push` low value order event to data layer if it is not returned in request header', () => {
```

Comment:
Should this test that the dataLayer wasn't called instead of checking for an empty array?